frontend/src/Constants/ErrorMessages.js

```diff
@@ -6,7 +6,7 @@ export const PASSWORD_AT_LEAST_ONE_LOWERCASE =
 export const PASSWORD_AT_LEAST_ONE_UPPERCASE =
   "Please input at least one upper case letter!";
 export const PASSWORD_AT_LEAST_ONE_SPECIAL_CHARACTER =
-  "Please input at least one special character!";
+  "Please input at least one special character without space and unsported special character!";
 export const PASSWORD_RANGE_FROM_8_TO_16_CHARACTERS =
```

Comment:
Typo: This string should be: Please input at least one special character without space and the unsupported special characters!
Please double check